### DISCUSSION TITLE
How to credit a partial distribution?

### DISCUSSION THREAD

**Discussion by gfireman** [CCSDSPy/ccsdspy#96] [General]
I'm implementing CCSDSPy for monitoring PACE OCI telemetry.  The code is not expected to be used by the public, but will be publicly available as part of our OCSSW code base.

I've found that we need only the subset ccsdspy/ccsdspy/*.py.  I'd rather include just those files than require that the entire package be installed for each of our many processors.  I was thinking of including a README file like:

```
This directory contains a subset of the complete CCSDSPy package, available at https://github.com/CCSDSPy/ccsdspy.
Credit:
Daniel da Silva, Steven Christe, Stefan Codrescu, J. Marcus Hughes, & Ezra Brooks. (2023). CCSDSPy (1.1.0). Zenodo.
https://doi.org/10.5281/zenodo.7819991
```
Does this meet your approval?  Any suggested changes?

**Comment by ddasilva** [comment]
Hi @gfireman, This looks acceptable to me. Thanks for giving us a mention!

Best,
Daniel

**Comment by gfireman** [comment]
Will do - thanks!
